lib-js/test-case/expected.ts

```diff
@@ -0,0 +1,66 @@
+import { SpecDirectory } from "../spec-directory"
+import { SassResult } from "./util"
+
+// Ensure that the directory has exactly one output or error file
+function checkDuplicateOutputs(dir: SpecDirectory, impl: string) {
+  if (dir.hasFile(`output-${impl}.css`) && dir.hasFile(`error-${impl}`)) {
+    throw new Error(`Found both output and error file for ${impl}.`)
+  }
+
+  if (dir.hasFile(`output.css`) && dir.hasFile(`error`)) {
+    throw new Error(`Found both \`output.css\` and \`error\` file.`)
+  }
+}
+
+function expectsSuccess(dir: SpecDirectory, impl: string) {
```

Comment:
This name is kind of confusing—it sounds like it's verifying that `dir` expects a successful outcome, but it will pass for directories that expect errors as well. Maybe `checkOutputExpectation`?

Reply:
It does — it returns `true` if it expects success, and `false` if it expects an error. It's not really a `check` because it's supposed to return a boolean, not an error. I've added a comment, but do you have a better name idea?